src/components/TypingIndicator/TypingIndicator.vue

```diff
@@ -0,0 +1,159 @@
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div v-show="typingParticipants.length" class="indicator">
+		<div class="indicator__wrapper">
+			<div class="indicator__avatars">
+				<AvatarWrapper v-for="(participant, index) in visibleParticipants"
+					:id="participant.actorId"
+					:key="index"
+					:name="participant.displayName"
+					:source="participant.actorType"
+					small
+					condensed
+					:condensed-overlap="8"
+					disable-menu
+					disable-tooltip />
+			</div>
+			<p class="indicator__main" v-html="indicatorMessage" />
+		</div>
+	</div>
+</template>
+
+<script>
+import AvatarWrapper from '../AvatarWrapper/AvatarWrapper.vue'
+
+export default {
+	name: 'TypingIndicator',
+	components: { AvatarWrapper },
+
+	props: {
+		/**
+		 * The conversation token
+		 */
+		token: {
+			type: String,
+			required: true,
+		},
+	},
+
+	computed: {
+		participants() {
+			return this.$store.getters.participantsList(this.token).filter(participant => {
+				return participant.actorId !== this.$store.getters.getActorId()
+			})
+		},
+
+		typingParticipants() {
+			// TODO implement signal receiving here
+			return this.participants
+		},
+
+		visibleParticipants() {
+			return this.typingParticipants.slice(0, 3)
+		},
+
+		hiddenParticipantsCount() {
+			return this.typingParticipants.slice(3).length
+		},
+
+		indicatorMessage() {
+			if (!this.typingParticipants) {
+				return ''
+			}
+
+			const [user1, user2, user3] = this.prepareNamesList()
+
+			if (this.typingParticipants.length === 1) {
+				return t('spreed', '{user1} is typing...',
+					{ user1 }, undefined, { escape: false })
+			}
+
+			if (this.typingParticipants.length === 2) {
+				return t('spreed', '{user1} and {user2} are typing...',
+					{ user1, user2 }, undefined, { escape: false })
+			}
+
+			if (this.typingParticipants.length === 3) {
+				return t('spreed', '{user1}, {user2} and {user3} are typing...',
+					{ user1, user2, user3 }, undefined, { escape: false })
+			}
+
+			return n('spreed', '{user1}, {user2}, {user3} and %n other are typing...',
+				'{user1}, {user2}, {user3} and %n others are typing...',
+				this.hiddenParticipantsCount, { user1, user2, user3 }, { escape: false })
+		},
+	},
+
+	methods: {
+		prepareNamesList() {
+			return this.visibleParticipants.reverse()
+				.map(participant => this.getParticipantName(participant))
+				.map(name => name ? `<strong>${name}</strong>` : undefined)
```

Comment:
you need to escape the names here

Reply:
Good point 
![image](https://user-images.githubusercontent.com/93392545/233624069-bef03778-a3a5-42aa-ae13-f27d796d8423.png)

Reply:
> Good point. But shouldn't it be forbidden to set a name with these special characters?

Too late and therefore no.